tests/acceptance/new-test.js

```diff
@@ -285,6 +287,14 @@ describe('Acceptance: ember new', function() {
   }));
 
   it('ember new uses yarn when blueprint has yarn.lock', co.wrap(function *() {
+    try {
+      let yarn = new Command('yarn');
```

Comment:
running `yarn` might be problematic, as it would actually run and install stuff if it actually is available. we might be able to use something like https://github.com/npm/node-which to figure out if `yarn` exists or not 🤔